plugins/deepgram/src/stt.ts

```diff
@@ -261,12 +261,12 @@ export class SpeechStream extends stt.SpeechStream {
                   if (isFinal) {
                     this.queue.put({
                       type: stt.SpeechEventType.FINAL_TRANSCRIPT,
-                      alternatives: [alternatives[0], ...alternatives.splice(0)],
```

Comment:
This just duplicates the first item in the array. Not sure what the intention was, perhaps to shallowly copy the array?